content/docs/gto/user-guide.md

```diff
@@ -0,0 +1,109 @@
+# User Guide
+
+GTO helps you build a Artifact Registry out of your Git repository. It does so
+by creating Git tags of special format and managing `artifacts.yaml` metafile.
+Since committing large files to Git is not a good practice, you should consider
+not committing your artifacts to Git. Instead, use [DVC](https://dvc.org/doc),
+Git-lfs, or any method commit pointers to the binary files instead.
+
+## Annotations in artifacts.yaml
+
+Using Git tag to register versions and assign stages is handy, but the Git tag
+itself doesn't contain path to the artifact, type of it (it could be `model` or
+`dataset`), or any other information you may find useful. For simple projects
+(e.g. single artifact) we can assume the details in a downstream system. But for
+more advanced cases, we should codify them in the registry itself.
```

Comment:
```suggestion
## Annotations in artifacts.yaml

Using Git tag to register versions and assign stages is handy, but the Git tag
itself doesn't contain path to the artifact, type of it (it could be `model` or
`dataset`), or any other information you may find useful. For simple projects
(e.g. single artifact) we can assume the details
[in a CI/CD system](#acting-in-ci-cd) downstream. But for more advanced cases,
we should codify them in the registry itself.
```